pytest_django/fixtures.py

```diff
@@ -180,8 +180,16 @@ class PytestDjangoTestCase(test_case_class):  # type: ignore[misc,valid-type]
 
 
 def _disable_native_migrations() -> None:
+    # https://github.com/django/django/commit/f5ebdfce5c417f9844e86bccc2f12577064d4bad
+    if get_django_version() >= (3, 1):
+        from django.db import connections
+        for alias in connections:
+            connection = connections[alias]
+            test_settings = connection.settings_dict['TEST']
+            test_settings['MIGRATE'] = False
+        return
```

Comment:
The `MigrateSilentCommand` stuff still seems relevant, for silencing the `migrate` output.

Reply:
I thought that the whole point of `_disable_native_migrations()` is that the migrations are disabled. Why would you want to silence them? Isn't the purpose of the `--nomigrations` flag to prevent migrations from running? If so, the the `migrate` command shouldn't run and there should be no output to suppress. What does the word `native` mean in this context? Is this only supposed to suppress some migrations but not all?